README.md

```diff
@@ -48,6 +48,7 @@ See [these examples](example) for more information
   type: Chart.ChartType
   data: Chart.ChartData | (canvas: HTMLCanvasElement | null) => Chart.ChartData;
   options?: Chart.ChartOptions;
+  accessibilityOptions?: {ariaLabel: string, fallbackContent: React.ReactNode};
```

Comment:
I don't think these need to be coupled together.

Reply:
I've just updated that now, fallbackContent is an optional prop and I've extended AriaAttributes on the interface so we can just pass any aria attributes straight down to the canvas. I don't foresee any issues with this but let me know if I've missed anything.